go.mod

```diff
@@ -19,27 +19,41 @@ require (
 )
 
 require (
+	github.com/CMSgov/bcda-app v0.0.0-20240116200610-366105d4b80b // indirect
```

Comment:
We should remove this dependency and update the go.mod and go.sum files, based on my comment [here](https://github.com/CMSgov/bcda-ssas-app/pull/151/files#r1467953507)!